components/places/uniffi.toml

```diff
@@ -0,0 +1,5 @@
+[bindings.kotlin]
+# Typically there'd be no `.uniffi` tail, but without it we see obscure
+# kotlin errors about being unable to resolve which `rustCall` to use.
```

Comment:
Please leave an additional note here along the lines of "this can go away once we UniFFI the entire places crate".